src/App.js

```diff
@@ -1,5 +1,28 @@
+import Header from "./components/header/Header";
+// import OrderDetails from "./components/orderdetail/OrderDetails";
+// import MakeBurger from "./components/makeburger/MakeBurger";
+import BurgerBuilder from "./layout/burgerBuilder";
+import Auth from "./components/auth/Auth";
+import { Routes, Route } from "react-router-dom";
+import Checkout from "./components/checkout/Checkout";
+import MakeBurger from "./components/makeburger/MakeBurger";
+import Order from "./components/order/order";
 function App() {
-  return <>Google</>;
+  return (
+    <>
+      <Header />
+
+      <Routes>
```

Comment:
we can also create an array of objects of the form `{path, Element}` and then map Route onto it.